extensions/toggl-track/src/toggl/index.ts

```diff
@@ -1,72 +1,72 @@
-import { Client, Project, Tag, TimeEntry, Workspace } from "./types";
+import { Client, Me, Project, Tag, TimeEntry, Workspace } from "./types";
 import { getPreferenceValues } from "@raycast/api";
 import { authenticatedFetch } from "./auth";
 
 const TogglAPI = function (apiToken: string) {
-  const baseUrl = "https://api.track.toggl.com/api/v8";
+  const baseUrl = "https://api.track.toggl.com/api/v9";
   const api = authenticatedFetch(apiToken, baseUrl);
 
   return {
+    getMe: (): Promise<Me> => {
+      return api.get<Me>("/me");
+    },
     getWorkspaces: (): Promise<Workspace[]> => {
       return api.get<Workspace[]>("/workspaces");
     },
     getWorkspaceProjects: async (workspaceId: number): Promise<Project[] | null> => {
       const projects = (await api.get<Project[] | null>(`/workspaces/${workspaceId}/projects`)) || [];
       projects.push({
         id: -1,
-        wid: workspaceId,
-        cid: -1,
+        workspace_id: workspaceId,
+        client_id: -1,
         name: "No project",
         billable: false,
-        is_private: false,
-        active: true,
-        template: false,
-        at: new Date(),
-        created_at: new Date(),
         color: "",
-        auto_estimates: false,
-        actual_hours: 0,
-        hex_color: "",
       });
       return projects;
     },
     createTimeEntry: ({
       projectId,
+      workspaceId,
       description,
       tags,
       billable,
     }: {
       projectId?: number;
+      workspaceId: number;
       description: string;
       tags: string[];
       billable: boolean;
     }) => {
-      return api.post<{ data: TimeEntry }>(`/time_entries/start`, {
-        time_entry: {
-          description,
-          pid: projectId !== -1 ? projectId : undefined,
-          billable,
-          tags,
-          created_with: "raycast-toggl-track",
-        },
+      const now = new Date();
+
+      return api.post<{ data: TimeEntry }>(`/workspaces/${workspaceId}/time_entries`, {
+        billable,
+        created_with: "raycast-toggl-track",
+        description,
+        // For running entries should be -1 * (Unix start time)
+        duration: Math.floor((-1 * now.getTime()) / 1000),
+        project_id: projectId !== -1 ? projectId : undefined,
```

Comment:
If you change according to comments about `-1` elsewhere, we can change this to:
```suggestion
        project_id: projectId,
```